docs/getting_started/installation.md

```diff
@@ -98,19 +98,88 @@ set any of them.
 
 #### Required in production:
 
-* `SERVER_API_TOKEN` - token for authorizing HTTP requests. Defaults to `development` for
-development builds.
-* `SECRET_KEY_BASE` - used to sign/encrypt tokens generated for Peers.
-* `VIRTUAL_HOST` - host passed to the endpoint config. Defaults to `localhost` for development builds.
-* `PORT` - port to run the HTTP server on. Defaults to `5002` for development builds.
+* `JF_SERVER_API_TOKEN` - token for authorizing HTTP requests.<br/>
+Defaults to `development` for development builds.
+* `JF_HOST` - defines how Jellyfish should be seen from the outside.<br/>
+Defaults to `JF_IP:JF_PORT`.<br/>
+It can be in the form of `ip:port`, `domain:port` or simply `domain`.
+Useful when hosting Jellyfish behind proxy.
+It is returned e.g. when creating a new room.
+When running with Docker, `JF_IP` is set to `0.0.0.0`
+making the default value of `JF_HOST` incorrect.
+Therefore, for Docker, you have to set `JF_HOST` manually.
+For running Docker locally, it can simply be `localhost:8080`.
 
 #### Optional:
```

Comment:
Consider adding extra separator header before the distribution variables. IMO it looks nicer and easier to read.
The required/ optional are now h3 headers (###) and distribution variables are h4

<img width="1025" alt="image" src="https://github.com/jellyfish-dev/jellyfish-docs/assets/48837433/5e7819c4-1a2b-46cd-8a6e-ffd9bae20e64">

Reply:
Nice, will do!